example/rust/src/bin/protosocket.rs

```diff
@@ -0,0 +1,76 @@
+use log::info;
+use momento::{CredentialProvider, MomentoError, ProtosocketCacheClient};
+use std::process;
+use std::time::Duration;
+use tokio_rustls::rustls::crypto::aws_lc_rs::default_provider;
+
+#[tokio::main]
+async fn main() -> Result<(), MomentoError> {
+    tokio_rustls::rustls::crypto::CryptoProvider::install_default(default_provider())
+        .expect("Error installing default crypto provider");
+
+    env_logger::init();
+    info!("Starting Momento ProtosocketCacheClient example");
+
+    let credential_provider = CredentialProvider::from_env_var("MOMENTO_API_KEY".to_string())
+        .expect("auth token should be valid");
+    // .with_private_endpoints(); // use this if you want to connect to private endpoints instead of public ones
+    // .secure_endpoint_override("127.0.0.1:9004"); // use this if you know the exact address to connect to already
```

Comment:
@anitarua maybe remove this line? 
```
    // .secure_endpoint_override("127.0.0.1:9004"); // use this if you know the exact address to connect to already
 ```

Is there ever reason external customer would want to do this or is this just for us testing internally?